README.md

```diff
@@ -276,6 +276,22 @@ If key generation didn't work or you want to use a custom key and certificate, y
 pyrdp-mitm.py 192.168.1.10 -k private_key.pem -c certificate.pem
 ```
 
+##### Monster-in-the-Middle NLA/CredSSP connections
+You can also specify the server's key and certificate along with the
+authentication as `ssp` to start an NLA connection against the server:
+```
+--auth ssp
+```
+This will enable the possibility to intercept NLA enabled connections.
+
+##### Alternative host redirection
+If the server you are trying to connect enforces NLA, you can redirect the
+connection to another non-NLA server to keep the connection alive. To enable
+this feature append specify the alternative host's address and port:
+```
+--nla-redirection-host 192.168.1.12 --nla-redirection-port 3389
+```
```

Comment:
Add a diagram, I will also use it in the blog post since this was added in 1.1.0 which is covered by my next blog